src/components/CTable.vue

```diff
@@ -29,19 +29,29 @@
           </tr>
         </thead>
         <tbody>
-          <c-row
-            v-for="m in displayedMessages"
-            :key="m.messageId"
-            :messageId="m.messageId"
-            :actorName="m.actorName"
-            :priority="m.priority"
-            :name="m.name"
-            :args="m.args"
-            :kwargs="m.kwargs"
-            :enqueuedDatetime="m.enqueuedDatetime"
-            :startedDatetime="m.startedDatetime"
-            :endDatetime="m.endDatetime"
-          ></c-row>
+          <template v-for="m in displayedMessages">
+            <c-row
+              :key="m.messageId"
+              :messageId="m.messageId"
+              :actorName="m.actorName"
+              :priority="m.priority"
+              :name="m.name"
+              :opened="opened"
+              :progress="m.progress"
+              :args="m.args"
+              :kwargs="m.kwargs"
+              :enqueuedDatetime="m.enqueuedDatetime"
+              :startedDatetime="m.startedDatetime"
+              :endDatetime="m.endDatetime"
+              @onToggle="toggle"
+            ></c-row>
+            <tr :key="m.messageId" v-if="opened.includes(m.messageId)" :colspan="columns.length">
+              <td class="border px-4 py-2">
+                <pre class="text-xs whitespace-normal">{{ m.args }}</pre>
```

Comment:
Args: {{ m.args }}
Kwards: {{ m.kwargs }}
And add a small margin to the left of these fields.